less/window.less

```diff
@@ -83,6 +83,7 @@ html,
     height: 100%;
     width: 100%;
     &.isPreview {
+      will-change: opacity;
```

Comment:
nice find! There could be more elements which can be optimized with `will-change`.